docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -13,21 +13,25 @@ The following procedure applies to a single node standalone {OpenStackPreviousIn
 [subs=+quotes]
 ----
 ifeval::["{build}" != "downstream"]
+CONTROLLER1_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@<controller-1 IP>"
+# ...
+endif::[]
+ifeval::["{build}" == "downstream"]
+CONTROLLER1_SSH="ssh -i <path_to_SSH_key> root@<controller-1 IP>"
+# ...
+endif::[]
+# ... <1>
+ifeval::["{build}" != "downstream"]
 EDPM_PRIVATEKEY_PATH="~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa"
 endif::[]
 ifeval::["{build}" == "downstream"]
-EDPM_PRIVATEKEY_PATH="<path_to_SSH_key>"
+EDPM_PRIVATEKEY_PATH="<path_to_SSH_key>" <2>
 endif::[]
-declare -A computes
-computes=(
-  ["standalone.localdomain"]="192.168.122.100"
-  # ...
-)
 ----
 +
-** Replace `["standalone.localdomain"]="192.168.122.100"` with the name and IP address of the Compute node.
+<1> Complete `CONTROLLER<X>_SSH` settings with SSH connection details for all controllers, including cell controllers, of the source {OpenStackPreviousInstaller} cloud.
```

Comment:
@jistr this is still valid for a single cell, just that there is no cell controllers